tools/editorconfig/.editorconfig

```diff
@@ -0,0 +1,20 @@
+root = true
+
+[*]
+end_of_line = lf
+indent_style = space
+charset = utf-8
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{yml,twig,php}]
+indent_size = 4
```

Comment:
> but maybe being consistent with php files is better ?

Convention is convention. I don't see why we have to put 4 spaces everywhere because "PHP do it this way". :-)